### PR TITLE
docs: restructure

### PR DESCRIPTION
**Pull request by daKmoR**
- main content is now a "guide" it covers everything (so you can use it)
- extra section for `testing`, `linting`, `setup` and `faq`

=> should be easier to understand and if you want more details there are dedicated sections for it

Preview at: https://deploy-preview-121--nervous-stonebraker-1f429b.netlify.com/

### CODE REVIEW

**Comment by daKmoR** [comment]
ready for review/feedback :)

File: docs/guide/developing.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Developing with lit-html
+
+## Lit-html
+For developing web component we recommend using the `lit-html` library with the `LitElement` base class.
```
Comment (thepassle) [inline docs/guide/developing.md]: "We recommend using the `lit-html` library with the `LitElement` base class for developing web components."

File: docs/guide/developing.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Check out the official documentation for: [lit-html](https://lit-html.polymer-project.org/) and [lit-element](https://lit-element.polymer-project.org/)
+
+## Examples
+A collection of live code samples can be found on `Stackblitz`:
+
+* [Basic](https://stackblitz.com/edit/open-wc-lit-html-basics?file=README.md)
+
+## Other reading
```
Comment (thepassle) [inline docs/guide/developing.md]: "Further reading"

File: docs/guide/next-steps.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Next Steps
+
+Now that you have a basic knowledge howto work with webcomponents there are some more in depts informations.
```
Comment (thepassle) [inline docs/guide/next-steps.md]: "Now that you have a basic understanding of web components, you can find some more in depth information in these resources."

File: docs/linting/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Linting
+
+Please be sure that you finished [The Linting Guide](/guide/linting.html) before you move on.
```
Comment (thepassle) [inline docs/linting/README.md]: "Please make sure to finish [The Linting Guide](/guide/linting.html) before continuing."

File: docs/linting/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Linting
+
+Please make sure to finish [The Linting Guide](/guide/linting.html) before continuing.
+
+These are the more in depth informations for linting.
```
Comment (thepassle) [inline docs/linting/README.md]: These are some more in depth resources about linting.

File: docs/setup/README.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Setup
+
+Here you will find more detailed information about open-wc features.
```
Comment (thepassle) [inline docs/setup/README.md]: In this section you can find more information about open-wc features.

File: docs/testing/README.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+# Testing
+
+Please make sure to finish [The Testing Guide](/guide/testing.html) before continuing.
+
+These are the more in depth informations for testing.
```
Comment (thepassle) [inline docs/testing/README.md]: These are some more in depth resources for testing.

File: packages/testing/README.md [hunk 8]
```diff
@@ -71,7 +71,7 @@ Already part of `yo open-wc:testing`
   },
   ```
 
-For more details pls see [testing-karma](https://open-wc.org/recommendations/testing-karma.html)
+For more details pls see [testing-karma](https://open-wc.org/testing/testing-karma.html)
```
Comment (thepassle) [inline packages/testing/README.md]: For more details, please see [testing-karma](https://open-wc.org/testing/testing-karma.html)

File: packages/testing/README.md [hunk 9]
```diff
@@ -93,7 +93,7 @@ Already part of `yo open-wc:testing`
   },
   ```
 
-For more details pls see [testing-karma-bs](https://open-wc.org/recommendations/testing-karma-bs.html)
+For more details pls see [testing-karma-bs](https://open-wc.org/testing/testing-karma-bs.html)
```
Comment (thepassle) [inline packages/testing/README.md]: For more details, please see [testing-karma-bs](https://open-wc.org/testing/testing-karma-bs.html)

File: docs/about/README.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# About Us
+
+- We are programmers
+- We are webcomponent enthusiast
```
Comment (thepassle) [inline docs/about/README.md]: - We are web component enthusiasts

File: docs/about/README.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+- We are believers of Web Standards
+- We are knowledge hungry
```
Comment (thepassle) [inline docs/about/README.md]: - We are hungry for knowledge

File: docs/about/README.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+- We are open source
+- We are The Web
```
Comment (thepassle) [inline docs/about/README.md]: - We are the web

File: docs/about/README.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
+# About Us
+
+- We are programmers
+- We are webcomponent enthusiast
+- We are believers of Web Standards
```
Comment (thepassle) [inline docs/about/README.md]: - We are believers of web standards

File: docs/guide/developing.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
+# Developing with lit-html
```
Comment (LarsDenBakker) [inline docs/guide/developing.md]: I was doubting whether this should be called `developing with lit-html` or `developing web components` ?

**Comment by LarsDenBakker** [comment]
Looks good to me, placed one question.

File: docs/guide/developing.md (hunk 14 above)
Comment (daKmoR) [reply docs/guide/developing.md]: renamed to just `Developing`

**Pull request merged**